csharp/test/Ice/metrics/AllTests.cs

```diff
@@ -627,9 +627,8 @@ public static IMetricsPrx allTests(TestHelper helper, CommunicatorObserver obsv)
                 }
                 Thread.Sleep(10);
             }
-            test(cm1.Failures == 2 && sm1.Failures >= 2);
+            test(cm1.Failures == 1 && sm1.Failures >= 2);
```

Comment:
It's unclear why the exception renaming/refactoring changed this number.

Reply:
Seems like the invocation was causing a retry before, but not anymore.